data/processed/ref/2020_02_04_mean_ref.csv

```diff
@@ -0,0 +1,41 @@
+,year,nuts_id,nuts_year_spec,mean_ref
```

Comment:
I would remove the index column (which has no header anyway)

Reply:
Also I'd rename the file to jsut `mean_ref.csv` and I'd add a `data_date` field in the schema like:

```yaml
data_date: 20200204
```

Reply:
Thanks Luca! I will address these later :-)